src/components/Dialog/index.tsx

```diff
@@ -0,0 +1,58 @@
+import MuiDialog, { DialogProps as MuiDialogProps } from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import CloseIcon from '@mui/icons-material/Close';
+import {Button} from "../Button";
+import React from "react";
+export interface DialogTitleProps {
+    id: string;
+    children: React.ReactNode;
+    onClose: () => void;
+}
+
+export interface DialogProps extends MuiDialogProps {
+    handleClose:any;
+    title:string;
+    open: boolean;

```

Comment:
no need to explicitly mention 'open' prop, since it exists already in <Dialog> (https://mui.com/material-ui/api/dialog/#props)
'children' prop also seems redundant